config/sites/signage.sites.uiowa.edu/views.view.signage_groups.yml

```diff
@@ -86,6 +87,68 @@ display:
           multi_type: separator
           separator: ', '
           field_api_classes: false
+        field_group_email:
+          id: field_group_email
+          table: node__field_group_email
+          field: field_group_email
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: field
+          label: 'Group Email'
```

Comment:
This label needs to be changed to sentence case.  You can read more at https://docs.prod.drupal.uiowa.edu/development-workflow/content-type-best-practices.